diff --git a/src/libpisp/backend/meson.build b/src/libpisp/backend/meson.build
index e29e1a3..10ce7ac 100644
--- a/src/libpisp/backend/meson.build
+++ b/src/libpisp/backend/meson.build
@@ -4,7 +4,7 @@
 config_install_dir = get_option('prefix') / get_option('datadir') / 'libpisp'

 conf_data = configuration_data()
-conf_data.set('PISP_BE_CONFIG_DIR', '"' + config_install_dir + '"')
+conf_data.set('PISP_BE_CONFIG_DIR', '"./libpisp"')
 pisp_build_config = configure_file(output : 'pisp_build_config.h', configuration : conf_data)

 backend_sources = files([
